apps/storybook/src/NetworkPeople.stories.tsx

```diff
@@ -27,6 +27,10 @@ const peopleProps = (): ComponentProps<typeof NetworkPeople> => {
           role: 'Co-Investigator' as TeamRole,
         },
       ],
+      labs: [
```

Comment:
Its fine to have these hardcoded like this as you can alter them in a more granular component 👍